etherman/etherman_test.go

```diff
@@ -164,8 +164,10 @@ func TestSequencedBatchesEvent(t *testing.T) {
 	}, polygonzkevm.PolygonValidiumEtrogValidiumBatchData{
 		TransactionsHash: txsHash,
 	})
-	da.Mock.On("GetBatchL2Data", uint64(2), txsHash).Return(data, nil)
-	da.Mock.On("GetBatchL2Data", uint64(3), txsHash).Return(data, nil)
+	batchNums := []uint64{2, 3}
+	batchHashes := []common.Hash{txsHash, txsHash}
+	batchData := [][]byte{data, data}
+	da.Mock.On("GetBatchL2Data", batchNums, batchHashes, []byte{}).Return(batchData, nil)
 	_, err = etherman.ZkEVM.SequenceBatchesValidium(auth, sequences, auth.From, []byte{})
```

Comment:
```suggestion
	batchData := [][]byte{data, data}
	daMessage := common.HexToBytes("0x123456789123456789")
	da.Mock.On("GetBatchL2Data", batchNums, batchHashes, daMessage).Return(batchData, nil)
	_, err = etherman.ZkEVM.SequenceBatchesValidium(auth, sequences, auth.From, daMessage)
```

If you do something like this you will be testing that the same DA message that we're passing to the on-chain call is propagated to the DA function, which is exactly what we want to happen, easy win 🌈